artifacts/definitions/Windows/Detection/BinaryRename.yaml

```diff
@@ -81,6 +82,10 @@ sources:
       FROM glob(globs=TargetGlob)
       WHERE
         NOT IsDir AND NOT IsLink
-        AND ( lowcase(string=VersionInformation.OriginalFilename) in bins.Original
-            OR lowcase(string=VersionInformation.OriginalFilename) in bins.Internal )
-        AND NOT lowcase(string=Name) in bins.Filename
+        AND (
+            (( lowcase(string=VersionInformation.OriginalFilename) in bins.Original
+                OR lowcase(string=VersionInformation.InternalName) in bins.Internal )
+                AND NOT lowcase(string=Name) in bins.Filename )
+        OR FullPath =~ 'C:\\\\Windows\\\\System32\\\\(osk|Magnify|Narrator|DisplaySwitch).exe$'
```

Comment:
You could use OSPath.Basename here or is the intention to lock down the path as well?

Reply:
The intention is to be path specific to cover sticky keys executables.

The primary goal of this artifact is to discover renamed lolbin and other binaries of interest but this stickykeys usecase covers any binary that has replaced a normal sticky key binary.